License2Deploy/rolling_deploy.py

```diff
@@ -123,24 +124,19 @@ def get_lb(self):
   def calculate_autoscale_desired_instance_count(self, group_name, desired_state):
     ''' Search via specific autoscale group name to return modified desired instance count '''
     try:
-      cur_count = int(self.get_group_info(group_name)[0].desired_capacity)
       if desired_state == 'increase':
-        new_count = self.double_autoscale_instance_count(cur_count)
+        self.old_desired_capacity = int(self.get_group_info(group_name)[0].desired_capacity)
```

Comment:
No.  As mentioned above, `2` was set there just to satisfy a test case.  It could be set to 1000 as long as a number greater than 2.  It won't matter because this line right here will set it to the original desired capacity of the ASG.  It is 2 for dnbi, perhaps 3 for owl?  or 5 for whatever.  It is the ASG capacity that we later want to revert to.